Dependencies/AndroidExtensions/Include/AndroidExtensions/JavaWrappers.h

```diff
@@ -259,6 +261,18 @@ namespace android::content::res
     };
 }
 
+namespace android::graphics
+{
+    class SurfaceTexture : public java::lang::Object
+    {
+    public:
+        SurfaceTexture();
+
+        void initWithTexture(int texture);
```

Comment:
These wrappers are meant to map to the Java types 1:1, so I would expect a constructor that takes an `int texName`. Is there any reason you have a separate `initWithTexture` function?

Reply:
I first have to create a transition opengl context, create a texture within that context and init surface texture with that texture.

Reply:
I don't understand this. Can we sync up offline?

Reply:
+1